elyra/pipeline/validate.py

```diff
@@ -115,13 +115,15 @@ async def validate(self, pipeline: Dict) -> ValidationResponse:
 
         pipeline_execution = pipeline_json['pipelines'][0]['app_data'].get('runtime')  # local, kfp, airflow
         pipeline_runtime = self._get_runtime_schema(pipeline, response)
+        pipeline_source = pipeline_json['pipelines'][0]['app_data'].get('source')
```

Comment:
Do we still have/use source? Wasn't this changed as part of https://github.com/elyra-ai/elyra/issues/1773?

Reply:
as of today, the payload that I receive still contains source
![image](https://user-images.githubusercontent.com/13340013/128044620-0f596b0d-8b03-4faa-b207-177d81acdc8d.png)

Reply:
`source` seems useful.  Is the pipeline filename present under any other field?